### ISSUE TITLE
Navigation Widget

### ISSUE THREAD

**Issue by juliotaylor** [bigspring/monolith#183]
A navigation / menu widget that will allow us to choose where a nav tree starts and ends.

For example, if you had a menu with 3 levels of depth, we could add a widget that will start at say level 1 and end in level 2, allowing greater flexibility.

**Comment by juliotaylor** [comment]
@jrgmartin we discussed this recently. Did we say it's doable?

**Comment by jrgmartin** [comment]
Yes, this is definitely do-able.  The walker class supplies parameters for depth so we can limit it through there, and it should be fairly easy too.  We can hopefully modify the existing walker and just ignore it if the depth doesn't match what defined in the widget.
